src/models/room-state.ts

```diff
@@ -53,13 +53,15 @@ enum OobStatus {
 export interface IPowerLevelsContent {
     users?: Record<string, number>;
     events?: Record<string, number>;
+    notifications?: Record<string, number>;
```

Comment:
This type would be better specifying the known fields IMO

Reply:
looking at https://spec.matrix.org/v1.9/client-server-api/#mroompower_levels looks like `room` is the only field supported atm so i added that